topics/considerations-sharing-repo.adoc

```diff
@@ -0,0 +1,26 @@
+[[ccg-considerations-sharing-repo]]
+= Considerations for sharing a repository with engineering
+
+If you are considering sharing a repository with engineering, that is, for the software code and documentation to reside in the same upstream repository, here are a few things you will want to keep in mind.
+
+.Space considerations
+The size of the combined repository will be large, so if limited space is an issue, keep this factor in mind.
+
+.Rebase often
+When working in the same repository as engineering, rebasing frequently is often a necessity because engineering is making changes too. Before making any changes to the documentation you’ll want to rebase.
+
+.Upstream shared repository branches and downstream docs repository branches
+Although you’ll be sharing the upstream repository with engineering, you’ll also need a downstream docs repository. Since more than likely the upstream repository branching structure will not match exactly the downstream docs repository branching, you will need to sync up the content. The easiest way to sync up content is to use the git export script.  
+
+Also, figuring out a branching strategy ahead of time will save plenty of headaches in the long run.
+
+.Use the git export script 
+Another tip is to use the git export script to sync up content:
+https://github.com/redhat-documentation/tools/tree/master/git-export 
+
+.Use attributes
+In general, using attributes will facilitate ease of use between upstream docs and downstream docs. One of the most useful is for the product name itself: {ProductName}. Other useful attributes include those for any version numbers used in the docs or any other product names.  For more general information about how to use attributes, see https://redhat-documentation.github.io/community-collaboration-guide/#ccg-asciidoc-attributes.
+
+In particular, using a variable for the upstream directory in the downstream docs repo is helpful. For example, define the following variable in the downstream docs repository: `:UpstreamDir: upstream/documentation`
+Then, in the downstream master.adoc files, the include looks like this: `include::{UpstreamDir}/assemblies/assembly-installing.adoc[leveloffset=+1]`
```

Comment:
`:UpstreamDir: upstream/documentation` and `include::{UpstreamDir}/assemblies/assembly-installing.adoc[leveloffset=+1]` should be separate code blocks. 

Separate paragraphs with a blank line.